index.html

```diff
@@ -1,29 +1,69 @@
     <title>APE Festival Webpage</title>
     <link rel="stylesheet" href="./style.css" />
     <script defer src="./js/artists.js"></script>
+    <script defer src="./js/popup.js"></script>
+    <script defer src="./js/navigation.js"></script>
+    <script src="https://kit.fontawesome.com/76eab5b41f.js" crossorigin="anonymous"></script>
 </head>
+
 <body>
+    <header>
+        <div id="header-1">
+            <ul>
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <div id="header-2">
+            <aside id="popup">
+                <ul id="popup-list">
+                    <li>About</li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>Home</li>
+                </ul>
+            </aside>
+            <i class="fa-solid fa-bars" id="bars-button"></i>
+            <img src="./images/logo.jpg" id="header-img" />
+            <ul id='desktop-menu'>
+                <li>About</li>
+                <li>Program</li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li>Home</li>
+            </ul>
+        </div>
+    </header>
+
     <main>
-        <div id="header-text">
-            <h1 class="title bg">ALL POINTS<br />
+        <div id="header-wrapper">
+            <img id="stage" src="./images/stage.jpg" alt="Stage photo" />
+            <h1 id="title">ALL POINTS<br />
                 EAST FEST<br />
                 2023</h1>
-        </div>
-        <p id="event-desc">
-            All Points East is an annual music festival held over two weekends in
-            London's Victoria Park, run by AEG Presents. The 10-day event comprises
-            one festival weekend, four days of free entry and community activities
-            known as "In the Neighbourhood" and finally three standalone headline
-            shows in their APE Presents series. 
-        </p>
-        <div id="date-address">
-            <p id="date">2023.08.18(FRI) ~ 28(MON)</p>
-            <address>@ Victoria Park, London</address>
+            <p id="event-desc">
+                All Points East is an annual music festival held over two weekends in
+                London's Victoria Park, run by AEG Presents. The 10-day event comprises
+                one festival weekend, four days of free entry and community activities
+                known as "In the Neighbourhood" and finally three standalone headline
+                shows in their APE Presents series.
+            </p>
+            <div id="date-address">
+                <p id="date">2023.08.18(FRI) ~ 28(MON)</p>
+                <address>@ Victoria Park, London</address>
+            </div>
         </div>
```

Comment:
- [ ] It is required to stick to the given template as much as possible. You're only allowed to customize to content of the guiding template, but the layout and the number of elements on each of the two pages remain. Number of content on on this section does not correspond to the number of content we have on the guiding template. Kindly look into the two screenshots, compare the guiding template with your design, and walk your design alongside the guiding template.

### Your Design
![your-home-desktop-header](https://github.com/Movgto/capstone_1_project/assets/26597432/ca0206b6-f1f8-48bc-a800-8c5758c6ba40)

### Template
![home-desktop-header](https://github.com/Movgto/capstone_1_project/assets/26597432/c5cba77b-0ab4-49e3-bbfb-4ccaf9412358)



- [ ]  Same thing goes for the mobile version. Kindly walk your design alongside the guiding template

### Your Design
![your-home-mobile-header](https://github.com/Movgto/capstone_1_project/assets/26597432/0e1aedad-53a9-46a3-be47-0522a4f2f6d5)

### Template
![temp-home-mobile-header](https://github.com/Movgto/capstone_1_project/assets/26597432/15bf154d-e125-43c5-b0dc-76359a8b463a)